index.bs

```diff
@@ -12750,6 +12750,43 @@ The {{VoidFunction}} [=callback function=]
 type is used for representing function values that take no arguments and do not
 return any value.
 
+<h3>Locatlizable dictionary</h3>
+
+A localizable member is a dictionary member that represents a bidirectional algorithm paragraph when displayed, as defined by the bidirectional algorithm’s rules P1, P2, and P3, including, for instance, supporting the paragraph-breaking behavior of U+000A LINE FEED (LF) characters.
+
+A user agent is expected to honor the Unicode semantics of localizable members.
+
+The name of a localizable member is left to the defining specification (e.g., "title").
+
+Dictionaries that specify a localizable member must inherit from the {{Localizable}} dictionary.
+
+Specification authors must specify in prose which member(s) of the inheriting dictionary serve as localizable members. It is RECOMMENDED that specification authors limit localizable member in the prototype chain of inherited dictionaries (ideally to one).
+
+<pre class="idl">
+enum TextDirection {
```

Comment:
Maybe this should be its own chapter if we're going to have this here? This might be useful on its own as well.